neo4j-inserter/insta_user-inserter/Dockerfile

```
@@ -1,10 +1,21 @@
-FROM golang:1.13 as builder
+FROM golang:1.12-alpine as builder
+RUN apk add --no-cache ca-certificates cmake make g++ openssl-dev git curl pkgconfig
+# clone seabolt-1.7.0 source code
+RUN git clone -b v1.7.2 https://github.com/neo4j-drivers/seabolt.git /seabolt
```

We used `v1.7.2` to try to make the build work, but since that wasn't the issue, let's use the newer version
```suggestion
RUN git clone -b v1.7.4 https://github.com/neo4j-drivers/seabolt.git /seabolt
```